src/signalrclient/hub_connection_impl.cpp

```diff
@@ -139,8 +143,28 @@ namespace signalr
             .append(message.serialize()));
     }
 
+    bool hub_connection_impl::invoke_callback(const web::json::value& message)
+    {
+        auto is_progress = message.has_field(_XPLATSTR("P"));

```

Comment:
Where I'm starting to see a lot of _XPLATSTR("P") and the like, maybe it would look cleaner to replace it with a descriptively named const? I'm not sure what the "P" or "I" means in the json message anyway.